src/components/HeroSlider.js

```diff
@@ -0,0 +1,74 @@
+import "../Home.css";
+import data from "../mocks/en-us/featured-banners.json";
+import Slide from "./Slide";
+import { FaArrowCircleLeft, FaArrowCircleRight } from "react-icons/fa";
+import styled from "styled-components";
+import { useState } from "react";
+const StyledHeroSlider = styled.div`
+  height: 32em;
+  background-color: #6d0b9d;
+  color: white;
+  .arrow {
+    color: #647a32;
+    z-index: 1000;
+    position: absolute;
+    cursor: pointer;
+    font-size: 5em;
+    bottom: 50%;
+  }
+  .left {
+    right: 0.2em;
+  }
+  .right {
+    left: 0.2em;
+  }
+
+  div.slider {
+    width: 100%;
+    height: 100%;
+    overflow-x: hidden;
+  }
+`;
+const HeroSlider = ({ children, props }) => {
```

Comment:
We must receive the items we will render as a prop rather than importing the json directly in our component i.e 
```js
const HeroSlider = ({slides}) =>{

} 
```